docker/misk-node/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:alpine
+
+WORKDIR /tmp/web/
+
+# Populate yarn cache by installing all @misk/ packages
+RUN yarn add \
+    --pure-lockfile \
+    --non-interactive \
+      @misk/common \
+      @misk/components \
+      @misk/dev \
+      @misk/tslint \
+  && rm -rf /tmp/web/node_modules \
+  && yarn global add webpack webpack-cli webpack-dev-server
+
+CMD tail -f /dev/null
```

Comment:
Guess this is needed because you go to have a command? Merging the two would solve this.

Reply:
Main purpose is to allow this image to be run as an arbitrary environment that someone can run with `-it` flags or `docker exec` arbitrary commands into without it dying between commands.